app/src/main/java/xyz/lilsus/papp/di/AppDependencies.kt

```diff
@@ -44,7 +45,13 @@ class AppDependencies(context: Context, private val applicationScope: CoroutineS
 
     fun createMainViewModel(walletRepository: WalletRepository?): MainViewModel {
         val payUseCase = PayInvoiceUseCase(walletRepository)
-        return MainViewModel(payUseCase)
+        val cameraController = CameraControllerFactory(context).createController()
+        val analyzer = QrCodeAnalyzer()
+        cameraController.setImageAnalysisAnalyzer(
+            Executors.newCachedThreadPool(),
```

Comment:
do I manually need to close/cancel anything ? or are tasks in this thread pool cleaned up properly..?